cluster/cluster.yaml

```diff
@@ -266,6 +266,11 @@ Resources:
           FromPort: 22
           IpProtocol: tcp
           ToPort: 22
+        # Allow the OpenTracing NLB to target these ports on all workers.
+        - CidrIp: 0.0.0.0/0
+          FromPort: 30443
+          IpProtocol: tcp
+          ToPort: 30444
```

Comment:
@vetinari this will add support for the lightstep NLB.

Reply:
By the way, is there any particular reason why is this not parameterised? Do we intend to deploy it to all clusters?